copier/cli.py

```diff
@@ -157,6 +157,11 @@ def __init__(self, executable: "PathLike[str]") -> None:
             "Allow templates with unsafe features (Jinja extensions, migrations, tasks)"
         ),
     )
+    skip_tasks = cli.Flag(
+        ["-S", "--skip-tasks"],
```

Comment:
I still think `-T` would be better, or maybe `-K` (uppercase because that usually means "negative" option). @copier-org/maintainers any opinion on this?

Reply:
`-T` seems best IMO, uppercase for negation and `T` for **t**asks (also in case we might have other skip flags in the future).

Reply:
PR updated 👍🏼